hyperv/tests/test_hyperv.py

```diff
@@ -3,21 +3,44 @@
 # Licensed under a 3-clause BSD style license (see LICENSE)
 import pytest
 
+from datadog_checks.base.constants import ServiceCheck
+from datadog_checks.dev.testing import requires_py2, requires_py3
 from datadog_checks.hyperv import HypervCheck
 from datadog_checks.hyperv.metrics import DEFAULT_COUNTERS
 
 
-def test_check(aggregator, instance_refresh, dd_run_check):
+@requires_py3
+def test_check(aggregator, dd_default_hostname, dd_run_check):
+    check = HypervCheck('hyperv', {}, [{}])
+    check.hostname = dd_default_hostname
+
+    # Run twice for counters that require 2 data points
+    dd_run_check(check)
+    dd_run_check(check)
+
+    aggregator.assert_service_check(
+        'hyperv.windows.perf.health', ServiceCheck.OK, count=2, tags=['server:{}'.format(dd_default_hostname)]
+    )
+    _assert_metrics(aggregator)
+
+
+@requires_py2
+def test_check_legacy(aggregator, instance_refresh, dd_run_check):
```

Comment:
Somewhat unrelated, I don't see `refresh_counters` in the pdh_legacy config template. Is this expected? Will the config spec be updated with the new perf_counters template later?